frontend/packages/ceph-storage-plugin/src/components/modals/volume-snapshot-modal/volume-snapshot-modal.tsx

```diff
@@ -195,7 +195,7 @@ export const VolumeSnapshotModal = withHandlePromise((props: VolumeSnapshotModal
         spec: {
           claimSelector: {
             matchLabels: {
-              thislabel: `schedule-${pvcLabel}`,
+              [pvcLabel]: `schedule-${pvcLabel}`,
```

Comment:
why is pvcLabel required? where is it coming from?

Are we still adding our labels and matching them?

Reply:
@cloudbehl this label is added to PVC for the schedule to trigger the creation of volume snapshot for that PVC. If we don't add this label, all PVC will have snapshots created according to this schedule. https://backube.github.io/snapscheduler/usage.html